test-data/unit/check-modules.test

```diff
@@ -670,45 +670,45 @@ def f(x: str) -> None: pass
 [case testTypeCheckNamedModule]
 # cmd: mypy -m m.a
 [file m/__init__.py]
-None + 1
```

Comment:
I only replaced the cases where the intent is to check whether mypy gets to a given line. The test basically assumes that `None + 1` will cause type error. But they don't fail because `--strict-optional` isn't the default. `object() + 1` will fail under any conditions, so I thought I'd use it.